frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -269,7 +279,16 @@ func toDispatchState(ctx context.Context, dt []byte, opt ConvertOpt) (*dispatchS
 		}
 
 		if v := st.Platform; v != "" {
-			v, u, err := shlex.ProcessWordWithMatches(v, metaArgsToMap(optMetaArgs))
+			result, err := shlex.ProcessWordWithMatches(v, metaArgsToMap(optMetaArgs))
+			v := result.Word
```

Comment:
nit: better to use the use properties directly I think. "result" could be "match(es)" or "m" if you like the short local vars.